### PR TITLE
SCMOD-3427: Create jep-talon base image

### PR DESCRIPTION
**Pull request by Anthony-McGreevy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,12 @@
-# opensuse-jeptalon-base
\ No newline at end of file
+# openSUSE and JepTalon image
+
+This project builds on the openSUSE Python2 image [here](https://github.com/CAFapi/opensuse-python2-image), Talon and Jep are then installed on the image. This image can be used as a base by projects which require Python2, Talon and Jep.
+
+### Tini
+[Tini](https://github.com/krallin/tini) is pre-installed in the container.  If the image entrypoint is not overwritten then it will be automatically used.
+
```
Comment (michael-bryson) [inline README.md]: If you are copying over the features from the python2 image readme can you also include the postgreSQL client section?

File: README.md [hunk 2]
```diff
@@ -1 +1,12 @@
-# opensuse-jeptalon-base
\ No newline at end of file
+# openSUSE and JepTalon image
+
+This project builds on the openSUSE Python2 image [here](https://github.com/CAFapi/opensuse-python2-image), Talon and Jep are then installed on the image. This image can be used as a base by projects which require Python2, Talon and Jep.
+
```
Comment (michael-bryson) [inline README.md]: Can you add a brief description of what Jep and Talon are as well as links to them?

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.github.cafapi</groupId>
+    <artifactId>opensuse-jeptalon-image</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <packaging>pom</packaging>
+
+    <name>openSUSE JepTalon image</name>
+    <description>An openSUSE-based image which includes jeptalon.</description>
+    <url>https://github.com/CAFapi/opensuse-python2-image</url>
```
Comment (michael-bryson) [inline pom.xml]: This url is incorrect

**Review by michael-bryson — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (michael-bryson) [reply README.md]: I have added this section along with some other feature information coming from the base image in this pull request: https://github.com/CAFapi/opensuse-jeptalon-base/pull/2

Comment (Anthony-McGreevy) [reply README.md]: PR Merged.

**Pull request merged**